File: tema-02/areaTriangulo.php
<!DOCTYPE html>
<!--
Escribe un programa que calcule el área de un triángulo.
-->
<html>
    <head>
        <title>Area Triangulo</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    </head>
    <body>
        <div>
            <h3>Area Triangulo</h3>
            <?php 
                $altura = $_GET["altura"];
                $base = $_GET["base"];
                $area = ($altura * $base) / 2;
                
                echo 'Resultado ', $area, ' metros cuadrados';
            ?>
        </div>
    </body>
</html>

File: capitolul3/src/capitolul3/AreaTriangulo.java
/*
Escribe un programa que calcule el área de un triángulo.
 */
package capitolul3;

import java.util.Scanner;

/**
 *
 * @author costy
 */
public class AreaTriangulo {
    public static void main(String[] args) {
        Scanner t = new Scanner(System.in);
        
        int base;
        int altura;
        
        System.out.print("Introduce la base del triangulo: ");
        base = t.nextInt();
        System.out.print("Introduce la altura del triangulo: ");
        altura = t.nextInt();
        
        double area = (double)(base * altura) / 2;
        System.out.println("El area del triangulo es de " + area + " cm cuadrados");
    }
}
